src/components/filmDescription/FilmDescription.js

```diff
@@ -23,16 +29,35 @@ const FilmDescription = ({film}) => {
           {overview}
         </p>
       </div>
+      <Button
+        id='closeDescription'
+        title='Close description'
+        disabled={false}
+        btnClass='btn--primary'
+        callback={() => {closeDescription()}}
+      />
     </div>
   );
 };
 
+function mapStateToProps(state) {
+  return {
+    selectedFilm: state.selectedFilm
+  };
+}
+
+function matchDispatchToProps(dispatch) {
+  return bindActionCreators({closeDescription}, dispatch)
+}
+
 FilmDescription.defaultProps = {
-  film: {},
+  selectedFilm: null,
+  closeDescription: null,
 };
 
 FilmDescription.propTypes = {
-  film: PropTypes.instanceOf(Object),
+  selectedFilm: PropTypes.instanceOf(Object),
```

Comment:
here you can use PropsTypes.shape to specify selectedFilm's fields

Reply:
Done